recipes/lightcurvelynx/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "lightcurvelynx" %}
+{% set version = "0.0.8" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/lightcurvelynx-{{ version }}.tar.gz
+  sha256: 210488f509fea25208a83dbf205a5bcb084dcf9757d1ae76df28c1e9b2be9559
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}.*
+    - setuptools >=62
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >={{ python_min }}
+    - astropy-base
+    - cdshealpix
+    - citation-compass >=0.0.3
+    - dust_extinction
+    - extinction >=0.4.7
+    - jax
+    - matplotlib-base
+    - mocpy
+    - nested-pandas
+    - numpy >2.0
```

There is no numpy 3, so this might be tough?

```suggestion
    - numpy >=2
```